### PR TITLE
Feat/add maxar

### PR DESCRIPTION
**Pull request by javiermanzano**
Created a PR for Maxar's NFT

### CODE REVIEW

**Review by prpatel05 — APPROVED** [review]

**Comment by prpatel05** [comment]
Looks like you have some merge conflicts, can you resolve so I can merge?

**Comment by javiermanzano** [comment]
Solved @prpatel05

File: src/cadence/testnet/getNFTs.cdc [hunk 1]
```diff
@@ -2998,23 +2999,22 @@ pub fun getProShop5(owner: PublicAccount, id: UInt64): NFTData? {
         metadata: {},
      )
 }
-     
-// https://flow-view-source.com/mainnet/account/0xa4e9020ad21eb30b/contract/QRLNFT
-pub fun getQRLNFT(owner: PublicAccount, id: UInt64): NFTData? {
```
[the remark this answers is not in the document]
Comment (javiermanzano) [reply src/cadence/testnet/getNFTs.cdc:3003]: Renabled. There was an error in the merge conflict

File: src/cadence/testnet/testGetNFTIDs.sh [hunk 2]
```diff
@@ -153,11 +153,13 @@ QRL)
   echo "QRL"
```
[the remark this answers is not in the document]
Comment (javiermanzano) [reply src/cadence/testnet/testGetNFTIDs.sh:153]: nope. THere was an error and QRL should not be removed

**Comment by javiermanzano** [comment]
I think that everything is already fixed @prpatel05

**Comment by javiermanzano** [comment]
@prpatel05 could we please merge today? Thanks!

**Pull request merged**